go.mod

```diff
@@ -20,10 +20,12 @@ require (
 	github.com/floren/o365 v0.0.1
 	github.com/fsnotify/fsnotify v1.4.9
 	github.com/go-ole/go-ole v1.2.4 // indirect
+	github.com/gobwas/glob v0.2.3
 	github.com/google/go-write v0.0.0-20181107114627-56629a6b2542
 	github.com/google/gopacket v1.1.17
 	github.com/google/uuid v1.1.1
 	github.com/gravwell/gcfg v1.2.5
+	github.com/gravwell/ingest/v3 v3.3.12
```

Comment:
Can you run 'go mod tidy' and see if it gets rid of this? I don't see a reference to gravwell/ingest in your *code* so maybe it's just old stuff.

Reply:
go mod tidy appears to have cleaned all that garbage up.